pwa/generate-api.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+npm run generate-api
```

Comment:
Should we replace `npm run generate-api` in `package.json` with this script so there's no confusion/accidentally running the wrong script?